src/DevTools/MockRouter.tsx

```diff
@@ -1,6 +1,9 @@
 import { RouterConfig } from "Artsy/Router"
 import { buildClientApp } from "Artsy/Router/buildClientApp"
-import { createMockNetworkLayer } from "DevTools/createMockNetworkLayer"
+import {
+  createMockNetworkLayer,
+  createMockNetworkLayer2,
```

Comment:
What is the path forward / level of effort to unify this code around a single file?

Reply:
We need to find all usages of `mockResolvers` and replace with `mockData` and `mockMutationResults`. It can be quite fiddly and there may be edge cases we haven't accounted for yet. Also for stories you need to manually check that they still work, though it looks like there's only 5 story files using `mockResolvers` after this gets merged.

So I'd guess it's 1-3 days of work for a focused engineer, depending on how smoothly things go.